lib/Steps/Integrations/Electron.ts

```diff
@@ -4,17 +4,37 @@ import * as _ from 'lodash';
 import * as path from 'path';
 import { Args } from '../../Constants';
 import { exists } from '../../Helper/File';
-import { debug, green, l, nl, red } from '../../Helper/Logging';
+import { debug, dim, green, l, nl, red } from '../../Helper/Logging';
 import { SentryCli } from '../../Helper/SentryCli';
 import { BaseIntegration } from './BaseIntegration';
 
 const MIN_ELECTRON_VERSION_STRING = '1.7.0';
-// tslint:disable-next-line:radix
 const MIN_ELECTRON_VERSION = parseInt(
   MIN_ELECTRON_VERSION_STRING.replace(/\D+/g, ''),
+  10,
 );
+
+const CODE_EXAMPLE = `const Sentry = require('@sentry/electron');
+
+Sentry.init({
+  dsn: '___DSN___',
```

Comment:
Should we want use `___PUBLIC_DSN___` here as well?

Reply:
It's not replaced by the same mechanism, and TIL `___DSN___` will be changed to emit the public DSN soon, so I'm gonna leave it for brevity.